### PR TITLE
Project list visuals

### PR DESCRIPTION
**Pull request by MeBrei**
Partially solves #186

### CODE REVIEW

**Comment by MeBrei** [comment]
Follow ups:
- Number formatting (eg 1100 -> 1.1k) for `Stats`
- Remove `My projects` title and move `New Project` button into new top bar (also part of #186 )
- Use actual statistics data

File: ui/DesignSystem/Primitive/Icon/CheckScallop.svelte [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<script>
```
Comment (rudolfs) [inline ui/DesignSystem/Primitive/Icon/CheckScallop.svelte]: I changed my mind about the file name. I think we should call it "Badge.svelte".

## Badge:
Screenshot 2020-02-26 at 09 59 23

## vs scallop:

Screenshot 2020-02-26 at 09 59 38

Comment (MeBrei) [reply ui/DesignSystem/Primitive/Icon/CheckScallop.svelte]: 😃 you convinced me!

**Comment by MeBrei** [comment]
I talked to @julien about the project list styling. The stats should not be vertically aligned by a fixed width as it would cause a lot of whitespace. I already addressed some of his feedback:
- `li` line height should be fixed
- project description should be truncated

File: ui/DesignSystem/Component/Stat.svelte [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<style>
+  .stat {
+    display: flex;
+    align-items: center;
+  }
+</style>
+
+<div class="stat" {style}>
+  <svelte:component this={icon} style="margin-right: 4px;" />
+  <Numeric style="color: var(--color-gray);">{count}</Numeric>
```
Comment (rudolfs) [inline ui/DesignSystem/Component/Stat.svelte]: The number should be darkgray (icon is gray):

Screenshot 2020-02-27 at 14 22 20

**Pull request merged**